verkle-trie/src/database/generic.rs

```diff
@@ -30,7 +31,8 @@ impl<T: BatchWriter> WriteOnlyHigherDb for GenericBatchWriter<T> {
         let mut labelled_key = Vec::with_capacity(key.len() + 1);
         labelled_key.push(STEM_TABLE_MARKER);
         labelled_key.extend_from_slice(&key);
-        self.inner.batch_put(&labelled_key, &meta.to_bytes());
+        self.inner
+            .batch_put(&labelled_key, &meta.to_bytes().unwrap());
```

Comment:
Can you explain the unwrap seen here? I'm guessing its an incremental change and will be eventually removed also?

Reply:
Yes, so because all of our conversions to bytes are fallible (and shouldn't logically be ignored imo, since they can sometimes be handled, the app should fail though not the library) these unwraps must slowly move upward until they can make it to the consumer